financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkingsavetolinkverification/NetworkingSaveToLinkVerificationViewModel.kt

```diff
@@ -72,28 +82,39 @@ internal class NetworkingSaveToLinkVerificationViewModel @Inject constructor(
         )
         onAsync(
             NetworkingSaveToLinkVerificationState::confirmVerification,
+            onSuccess = {
+                saveToLinkWithStripeSucceeded.set(true)
+                goNext(Pane.SUCCESS)
+            },
             onFail = { error ->
+                saveToLinkWithStripeSucceeded.set(false)
                 logger.error("Error confirming verification", error)
                 eventTracker.track(Error(PANE, error))
+                goNext(Pane.SUCCESS)
```

Comment:
Should this be Pane.FAILURE or something?

Reply:
Regardless of the result, we always navigate to the success pane. Then there we'd show a warning that your account could not be networked (saved to Link), but it was still connected with the merchant. See the video in this PR: https://github.com/stripe/stripe-android/pull/6414